scrape/scrape.go

```diff
@@ -1508,11 +1507,7 @@ loop:
 				e.Ts = t
 			}
 			_, exemplarErr := app.AppendExemplar(ref, lset, e)
-			exemplarErr = sl.checkAddExemplarError(exemplarErr, e, &appErrs)
-			if exemplarErr != nil {
-				// Since exemplar storage is still experimental, we don't fail the scrape on ingestion errors.
-				level.Debug(sl.l).Log("msg", "Error while adding exemplar in AddExemplar", "exemplar", fmt.Sprintf("%+v", e), "err", exemplarErr)
-			}
+			exemplarErr = storage.CheckAppendExemplarError(sl.l, exemplarErr, e, targetScrapeExemplarOutOfOrder, &appErrs.numExemplarOutOfOrder)
```

Comment:
It doesn't look like the return value is used anywhere here or in remote, perhaps the function doesn't need to return anything right now? Or else make it clear the value is ignored with `_ = ...`.

Reply:
You are totally right, I have removed the return value from the function